connector/routingconnector/README.md

```diff
@@ -0,0 +1,106 @@
+| logs | logs | [development] |
+
+[Exporter Pipeline Type]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/connector/README.md#exporter-pipeline-type
+[Receiver Pipeline Type]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/connector/README.md#receiver-pipeline-type
+[Stability Level]: https://github.com/open-telemetry/opentelemetry-collector#stability-levels
+<!-- end autogenerated section -->
+
+Routes logs, metrics or traces based on resource attributes to specific pipelines using [OpenTelemetry Transformation Language (OTTL)](../../pkg/ottl/README.md) statements as routing conditions.
+
+## Configuration
+
+If you are not already familiar with connectors, you may find it helpful to first visit the [Connectors README].
+
+The following settings are available:
+
+- `table (required)`: the routing table for this connector.
+- `table.statement (required)`: the routing condition provided as the [OTTL] statement.
+- `table.pipelines (required)`: the list of pipelines to use when the routing condition is met.
+- `default_pipelines (optional)`: contains the list of pipelines to use when a record does not meet any of specified conditions.
+- `error_mode (optional)`: determines how errors returned from OTTL statements are handled. Valid values are `ignore` and `propagate`. If `ignored` is used and a statement's condition has an error then the payload will be routed to the default pipelines.  If not supplied, `propagate` is used.
+
+Example:
+
+```yaml
+receivers:
+    otlp:
+
+exporters:
+  jaeger:
+    endpoint: localhost:14250
+  jaeger/acme:
+    endpoint: localhost:24250
+  jaeger/ecorp:
+    endpoint: localhost:34250
+
+connectors:
+  routing:
+    default_pipelines: [traces/jaeger]
+    error_mode: ignore
+    table:
+      - statement: route() where attributes["X-Tenant"] == "acme"
+        pipelines: [traces/jaeger-acme]
+      - statement: delete_key(attributes, "X-Tenant") where IsMatch(attributes["X-Tenant"], ".*corp")
+        pipelines: [traces/jaeger-ecorp]
+
+service:
+  pipelines:
+    traces/in:
+      receivers: [oltp]
+      exporters: [routing]
+    traces/jaeger:
+      receivers: [routing]
+      exporters: [jaeger]
+    traces/jaeger-acme:
+      receivers: [routing]
+      exporters: [jaeger/acme]
+    traces/jaeger-ecorp:
+      receivers: [routing]
+      exporters: [jaeger/ecorp]
+```
+
+A signal may get matched by routing conditions of more than one routing table entry. In this case, the signal will be routed to all pipelines of matching routes.
+Respectively, if none of the routing conditions met, then a signal is routed to default pipelines.
+
+## Differences between the Routing Connector and Routing Processor
+
+- The connector will only route using [OTTL] statements which can only be applied to resource attributes. It does not support matching on context values at this time.
+- The connector must be a receiver in a minimum of two pipelines.
```

Comment:
This actually a limitation of connectors before https://github.com/open-telemetry/opentelemetry-collector/pull/7840 merged. Since that has been merged, and released, we no longer have that limitation. I've updated the PR.